endelig gjennom alt.. (#120)
DHP-stottepakke/pages: pages/dmp_supporting_info/support_03_legal_ethics.md

## About this chapter

> National legislation, institutional guidelines and international norms have implications for the handling of research data. Often there is a need to balance availability and openness with confidentially. Both researchers and research institutions are legally responsible for complying with legal and ethical requirements.

Author: denne kom dobbelt i min copy paste mellom editor og git

Further information:
* [Report How should we share research data? (2021)](https://www.forskningsradet.no/siteassets/publikasjoner/2021/how-should-we-share-research-data.v2.pdf) 

## Question-specific guidance

> ### Involved institution(s)

Reviewer: ```suggestion
### Involved institution(s)/partner(s)
```

* [d. Guidelines for Research Ethics in Science and Technology by The Norwegian National Committee for Research Ethics in Science and Technology (NENT)](https://www.forskningsetikk.no/en/about-us/our-committees-and-commission/nent/guidelines-nent/guidelines-for-research-ethics-in-science-and-technology/ "2024-07-17")
* [e. Ethical Principles for Medical Research Involving Human Subjects (Declaration of Helsinki)](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/ "2024-07-17")
* [f. Guidelines by The Norwegian National Research Ethics Committee for medical and health research (NEM)](https://www.forskningsetikk.no/en/guidelines/medical-and-health-research/guidelines-by-nem/ "2024-07-17")
* [g. Guidelines for Ethical Research on Human Remains by The Human Remains Committee](https://www.forskningsetikk.no/en/guidelines/human-remains/guidelines-for-research-ethics-on-human-remains/ "2024-07-17")
* [h. Convention on Human Rights and Biomedicine (Oviedo Convention)](https://www.coe.int/en/web/bioethics/oviedo-convention "2024-07-19")

> * [e. Ethical Guidelines for the Use of Animals in Research](https://www.forskningsetikk.no/en/guidelines/science-and-technology/ethical-guidelines-for-the-use-of-animals-in-research/ "2024-07-17")

Reviewer: ```suggestion
* [i. Ethical Guidelines for the Use of Animals in Research](https://www.forskningsetikk.no/en/guidelines/science-and-technology/ethical-guidelines-for-the-use-of-animals-in-research/ "2024-07-17")
```

	* [Privacy policy for the University of Bergen](https://www.uib.no/en/personaldata/130126/privacy-policy-university-bergen "2024-07-16")
* University of Oslo (UiO)
	* [Processing of personal data in research projects at UiO](https://www.uio.no/english/for-employees/support/privacy-dataprotection/personal-data-in-research.html/ "2024-10-30")
	* [Privacy and data protection at UiO](https://www.uio.no/english/for-employees/support/privacy-dataprotection/index.html "2024-10-30")
	* [Privacy declarations for UiO](https://www.uio.no/english/about/regulations/privacy-declarations/ "2024-10-30")
	* [UiO Quality assurance system for health and medical research](https://www.uio.no/english/for-employees/support/research/quality-system-for-health-research/index.html "2024-10-30")
* UiT The Arctic University of Norway (UiT)
	* [Processing of personal data in research projects](https://en.uit.no/research/ethics/art?p_document_id=754184 "2024-07-16")
	* [Data protection/ Privacy at UiT](https://en.uit.no/om/informationsecurity#kapittel_705927 "2024-07-16")

#### Personal privacy considerations

> The routines regarding processing of personal data differ between institutions.  It is the researchers responsibility to make yourself familiar with the requirements at your institution, if not listet in the prvious question, look for simiar gudiance from your institution. 

Reviewer: ```suggestion
The routines regarding processing of personal data differ between institutions.  It is the researchers responsibility to make oneself familiar with the requirements at their institution. If not listed in the previous question, look for similar guidance from your institution. 
```

> Processing of personal data affect choice of storage, and how open data can be archived or made available to others, these considerations are part of data management planning. If the data are to be used for future research this must be taken into account when legal basis for data processing is selected. Explicit information about archiving and future use should also go into the information to participants.

Reviewer: ```suggestion
Processing of personal data affect choice of storage, and how open data can be archived or made available to others, these considerations are part of data management planning. If the data are to be used for future research this must be taken into account when legal basis for data processing is selected. Explicit information about archiving and future use should also be included in the information to study participants.
```

* Other routines for privacy assessment:
	* [RETTE at UiB](https://rette.app.uib.no/)
	* [NTNU guidance for Data Protection Impact Assessment (DPIA)](https://i.ntnu.no/wiki/-/wiki/English/Data+protection+impact+assessment "2024-10-30")
	* [UiO Quality assurance system for health and medical research](https://www.uio.no/english/for-employees/support/research/quality-system-for-health-research/index.html "2024-10-30")

##### Personal privacy considerations step-by-step
These questions will guide you through relevant considerations concerning collecting/processing personal data in accordance with GDPR. 

Be aware that you may have to register processing of personal data in additional systems as required by your institution, you do not need to duplicate this process if you have already gone trough this.

#### How will you document participants information and/or consent?

Documenting permissions form study participants to share data is crucial to making research data available, and often difficult or impossible to obtain in retrospect. It is therefore essential to include information about planned data sharing in the participant information letter and/or consent forms. 

> Ideally, the permission should be requested in a granular way, with distinct permission to making data available. Unfortunately, guidance on this topic and examples of consent forms that are adapted to European legislation is not easily found, and the lines of what and when is possible is still being drawn up. Consult your [institutional research data management support services](/pages/dmp_supporting_info/support_00_local_disc.md#institutional-rdm-support-alphabetical-order) if you are uncertain about formulations or need examples of how sharing and reuse of data including information about people can be possible..

Reviewer: ```suggestion

Ideally, the permission should be requested in a granular way, with distinct permission to making data available. Unfortunately, guidance on this topic and examples of consent forms that are adapted to European legislation is not easily found, and the lines of what and when is possible is still being drawn up. Consult your [institutional research data management support services](/pages/dmp_supporting_info/support_00_local_disc.md#institutional-rdm-support-alphabetical-order) if you are uncertain about formulations or need examples of how sharing and reuse of data including information about people can be possible.
```

> Consult your local research data management support services if you are uncertain about formulations or need examples of how sharing and reuse of data including information about people can be possible.

Reviewer: ```suggestion
```

> Remember that the ethical requirement to obtain permission to data sharing from study participants is independent of the legal basis (Norwegian: behandlingsgrunnlag) for processing of personal data as required by GDPR. This difference will however not be evident for research participants and the term 'consent' (Norwegian: samtykke) is often used for both processes. The requirements for the two consents are different.

Reviewer: ```suggestion
Remember that the ethical requirement to obtain permission to data sharing from study participants is independent of the legal basis (Norwegian: behandlingsgrunnlag) for processing of personal data as required by GDPR. This difference will however not be evident for research participants and the term 'consent' (Norwegian: samtykke) is often used for both processes. However, the requirements for the two types of consent differ.
```

Applying public interest as legal basis for scientific research projects is advisable. For compliance with ethical guidelines, permission to archive the data for the purpose of future research, and possibly also educational purposes should be collected and documented.

##### Research project using 'consent' as legal basis
Consent is a possible legal basis for processing of personal data under GDPR. For processing of certain types of personal data, strict requirements apply and processing needs to be based on a condition in addition to the legal basis. Explicit consent is one such possible condition, which must be freely given, specific, informed and unambiguous (GDPR article 9, article 7, and recital 32).

> If consent is used as legal basis for processing of personal data consider asking for permission to archive the data for the purpose of future research, and possibly also educational purposes as part of both the informed and specific consent. Consult the links below, to se if you can use standard consent clauses, which can be reflected with machine readable metadata.

Reviewer: ```suggestion
If consent is used as legal basis for processing of personal data consider asking for permission to archive the data for the purpose of future research, and possibly also educational purposes as part of both the informed and specific consent. Consult the links below, to see if you can use standard consent clauses, which can be reflected with machine readable metadata.
```
